.storybook-s2/docs/Migrating.jsx

```diff
@@ -411,6 +411,11 @@ export function Migrating() {
           <li className={style({font: 'body', marginY: 8})}>Update placement prop to be have one value (i.e. Update <Code>placement="bottom left"</Code> to be <Code>placement="bottom"</Code>)</li>
         </ul>
 
+        <H3>TreeView</H3>
+        <ul className="sb-unstyled">
+          <li className={style({font: 'body', marginY: 8})}>Update content within <Code>TreeViewItem</Code> to be wrapped in <Code>TreeViewContentItem</Code></li>
```

Comment:
not sure if this is confusing and people might think we mean to replace TreeViewItem with TreeViewContentItem

Reply:
Doesn't S1 TreeView also have TreeViewContent?

Reply:
technically? it's a little weird because it's a change from beta to GA. so if people are migrating from a beta TreeView then they'll need to update but if they end up using the GA version, then there should be no differences i believe

Reply:
oh I see, maybe we should just make that more explicit in the note (aka note that this is a .6 -> .7 S2 TreeView change)? Or maybe move it to the S2 release notes instead of the v3-> S2 migration notes?

Reply:
yeah i think we could be more explicit about the versioning then. i'll update it

Reply:
like we could say something like this "If migrating from TreeView version 3.0.0-beta.3 or before, please do the following. Otherwise, no updates needed."

Reply:
that sounds good to me